circle.yml

```diff
@@ -111,7 +111,15 @@ database:
     - 'echo "Skipping database phase (already ran)."'
 test:
   override:
-    - case $CIRCLE_NODE_INDEX in 0) bundle exec rake lint:javascript circle:run_tests ;; *) bundle exec rake circle:run_ui_tests ;; esac:
+    - bundle exec rake lint:javascript:
+        parallel: true
+        timeout:
+            7200
+    - bundle exec rake circle:run_tests:
+        parallel: true
+        timeout:
+            7200
+    - bundle exec rake circle:run_ui_tests:
```

Comment:
No - `parallel: true` just means "this step is allowed to run on all containers, not just container zero." ([documentation](https://circleci.com/docs/1.0/parallel-manual-setup/#splitting-your-test-suite)) It's not very intuitive.  We'll be running these in sequence, in part because of memory concerns.